src/stactools/noaa_sst/constants.py

```diff
@@ -0,0 +1,27 @@
+from datetime import datetime
+from pyproj import CRS
+from pystac import Provider, ProviderRole, Link
+
+NOAA_SST_ID = "noaa-sst"
+SST_EPSG = 4326
+SST_CRS = CRS.from_epsg(SST_EPSG)
+LICENSE = "proprietary"
+lic_link = "https://github.com/stactools-packages/noaa-sst/tree/main/src/stactools/noaa_sst/license"
+"/data-license.txt"
+LICENSE_LINK = Link(rel="license",
+                    target=lic_link,
+                    title="Public Domain License - NOAA")
+SPATIAL_EXTENT = [-180.0, 90.0, 180.0, -90.0]
+TEMPORAL_EXTENT = [
+    datetime(1985, 1, 1),
+    None,
+]
```

Comment:
Do we know the values here?

Reply:
For which variable?

Reply:
Both look like defaults, but if that's the case, that's fine

Reply:
Yup thats the case :)